ui/src/app/settings/views/Scripts/ScriptsUpload.js

```diff
@@ -0,0 +1,189 @@
+import PropTypes from "prop-types";
+import React, { useCallback, useEffect, useMemo, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { Redirect } from "react-router";
+import { useDropzone } from "react-dropzone";
+import pathParse from "path-parse";
+
+import { messages } from "app/base/actions";
+import Card from "app/base/components/Card";
+import Form from "app/base/components/Form";
+import FormCardButtons from "app/base/components/FormCardButtons";
+import Row from "app/base/components/Row";
+import actions from "app/settings/actions";
+import selectors from "app/settings/selectors";
+
+const baseStyle = {
+  flex: 1,
+  display: "flex",
+  flexDirection: "column",
+  alignItems: "center",
+  padding: "20px",
+  borderWidth: 2,
+  borderRadius: 2,
+  borderColor: "#eeeeee",
+  borderStyle: "dashed",
+  backgroundColor: "#fafafa",
+  color: "#bdbdbd",
+  outline: "none",
+  transition: "border .24s ease-in-out"
+};
+
+const activeStyle = {
```

Comment:
These would become something like `scripts-upload--active` and use the classnames dep to apply the right thing.